ribs/emitters/_genetic_algorithm_emitter.py

```diff
@@ -0,0 +1,183 @@
+"""Provides the GeneticAlgorithmEmitter."""
+import numpy as np
+
+from ribs._utils import check_batch_shape, check_shape
+from ribs.emitters._emitter_base import EmitterBase
+from ribs.emitters.operators import _get_op
+
+
+class GeneticAlgorithmEmitter(EmitterBase):
+    """Emits solutions by using operator provided
+
+    If the archive is empty and ``self._initial_solutions`` is set, a call to
+    :meth:`ask` will return ``self._initial_solutions``. If
+    ``self._initial_solutions`` is not set, we operate on self.x0.
+
+
+    Args:
+        archive (ribs.archives.ArchiveBase): An archive to use when creating and
+            inserting solutions. For instance, this can be
+            :class:`ribs.archives.GridArchive`.
+        x0 (np.ndarray): Initial solution.
+        operator (external class): Operator Class from pymoo or pygad
```

Comment:
Right now operator is an external class like pymoo's GaussianMutation class. The geneticAlgorithmEmitter itself creates an OperatorBase instance, but this could change - Operators could be exposed to the user at which point the pymoo_gaussian operator could take in itself GaussianMutation instance.